src/functions.php

```diff
@@ -56,6 +87,10 @@ function parallel(callable ...$callbacks): array
         }
     }
 
+    if ($exceptions) {
+        throw new \RuntimeException('One or more exceptions were thrown in parallel');
```

Comment:
Should we throw a custom exceptions allowing to retrieve all inner exceptions?

Reply:
Let's see if someone has an use case for that, don't want to add too much here, it's mainly to fix a bug